<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VatsinatorWindow</class>
 <widget class="QMainWindow" name="VatsinatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Vatsinator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/imgs.qrc">
    <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="MainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="MapView" name="Map">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="renderHints">
       <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="MainGridLayout">
      <item row="0" column="5">
       <widget class="QLabel" name="PositionBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">N E</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="ClientsBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Clients:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ReplaceableWidget" name="Replaceable" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>13</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuVatsinator">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>199</y>
      <width>191</width>
      <height>234</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>&amp;Vatsinator</string>
    </property>
    <addaction name="ActionMetar"/>
    <addaction name="ActionRefresh"/>
    <addaction name="EnableAutoUpdatesAction"/>
    <addaction name="ActionHomeLocation"/>
    <addaction name="separator"/>
    <addaction name="ActionFlightList"/>
    <addaction name="ActionATCList"/>
    <addaction name="separator"/>
    <addaction name="ActionExit"/>
   </widget>
   <widget class="QMenu" name="MenuHelp">
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="ActionAbout"/>
   </widget>
   <widget class="QMenu" name="MenuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="ActionPreferences"/>
   </widget>
   <addaction name="MenuVatsinator"/>
   <addaction name="MenuSettings"/>
   <addaction name="MenuHelp"/>
  </widget>
  <action name="ActionExit">
   <property name="icon">
    <iconset theme="application-exit" resource="../resources/imgs.qrc">
     <normaloff>:/uiIcons/application-exit.png</normaloff>:/uiIcons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="ActionAbout">
   <property name="icon">
    <iconset resource="../resources/imgs.qrc">
     <normaloff>:/mainIcon/icon.png</normaloff>:/mainIcon/icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="ActionMetar">
   <property name="icon">
    <iconset theme="weather-few-clouds" resource="../resources/imgs.qrc">
     <normaloff>:/uiIcons/weather-clouds.png</normaloff>:/uiIcons/weather-clouds.png</iconset>
   </property>
   <property name="text">
    <string>METARs...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+M</string>
   </property>
  </action>
  <action name="ActionRefresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="view-refresh" resource="../resources/imgs.qrc">
     <normaloff>:/uiIcons/view-refresh.png</normaloff>:/uiIcons/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string notr="true">F5</string>
   </property>
  </action>
  <action name="ActionPreferences">
   <property name="icon">
    <iconset theme="preferences-system" resource="../resources/imgs.qrc">
     <normaloff>:/uiIcons/configure.png</normaloff>:/uiIcons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="ActionFlightList">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flight list...</string>
   </property>
  </action>
  <action name="ActionATCList">
   <property name="text">
    <string>ATC list...</string>
   </property>
  </action>
  <action name="EnableAutoUpdatesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable auto-updates</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
  </action>
  <action name="ActionHomeLocation">
   <property name="text">
    <string>Show home location</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReplaceableWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/replaceablewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>ui/graphics/mapview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/imgs.qrc"/>
 </resources>
 <connections/>
</ui>
